# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../chrpath/chrpath-keepgoing.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

taken from gentoo
http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/app-admin/chrpath/files/chrpath-keepgoing.patch?revision=1.1

--- chrpath-0.13/main.c.orig	2006-02-27 20:21:06.000000000 +0100
+++ chrpath-0.13/main.c	2006-02-27 21:07:16.027641500 +0100
@@ -57,6 +57,7 @@
   printf("   -r <path>|--replace <path>  Replace current rpath/runpath setting\n");
   printf("                               with the path given\n");
   printf("   -l|--list                   List the current rpath/runpath (default)\n");
+  printf("   -k|--keepgoing              Continue as much as possible after an error.\n");
   printf("   -h|--help                   Show this usage information.\n");
 #ifndef HAVE_GETOPT_LONG
   printf("\n *** The long options are not available on this platform");
--- chrpath-0.13/chrpath.1.orig	2004-09-19 10:35:44.000000000 +0200
+++ chrpath-0.13/chrpath.1	2006-02-28 10:06:28.187060500 +0100
@@ -22,6 +22,7 @@
 [ \fB-r\fP \fI<path>\fP |\  \fB--replace\fP \fI<path>\fP ]
 [ \fB-c\fP | \fB--convert\fP ]
 [ \fB-l\fP | \fB--list\fP ]
+[ \fB-k\fP | \fB--keepgoing\fP ]
 [ \fB-h\fP | \fB--help\fP ]
 \fI<program>\fP [ \fI<program>\fP ... ]
 .SH DESCRIPTION
